tokio-quiche/src/http3/driver/mod.rs

```diff
@@ -648,7 +657,21 @@ impl<H: DriverHooks> H3Driver<H> {
                         qconn, stream_id, headers, prio, false,
                     )
                     .inspect(|_| ctx.initial_headers_sent = true)
+                };
+
+                if res.is_err() && first_non_fully_flush_time.is_none() {
```

Comment:
It may make the code's intention clearer to only set this value if the error is transient, like `StreamBlocked`. Currently for example, we'll set this on a `MessageError`, but we'd never actually expect to flush the frame in that case

Reply:
Good point, I was being a bit lazy